docs/shop_administration/manage_users.md

```diff
@@ -1,13 +1,82 @@
 ---
-description: You can manage and view user accounts in your system, including their shop activity such as orders or active baskets.
+description: You can manage and view customers and organizations' accounts in your system, including their shop activities such as orders.
 ---
 
-# Users
+# Customer management [[% include 'snippets/experience_badge.md' %]]
 
-Users are displayed in the **Users** section of the Back Office.
+In the Back Office, you can manage members of your team,
+customers and organizations' accounts in your system, including their shop activities such as orders.
 
-![](img/users_menu.png)
+## Create a new company
 
- [[% include 'snippets/commerce_badge.md' %]] The **eCommerce** tab in User content view contains an overview of the user's orders and active baskets.
+To create a new company, go to **Members** -> **Companies** section.
+There you can view a list of companies you have access to,
+you can also edit them or create a new one by selecting **Create** button in the top right corner.
 
-![](img/user_detaild_view.png)
+![Companies section](img/cp_companies_section_back_office.png)
+
+To create a new company, you need to provide:
+
+- name
+- tax ID
+- customer group
+- sales representative
+- billing address
+
+Optionally, you can add a website and other contact details.
+
+![Create a new company account](img/cp_create_company_back_office.png)
+
+## Manage company
+
+Created companies can be managed from the Back Office in the **Members** -> **Companies** section.
+Each company has its own profile where you can find:
+
+- summary with basic information and order history
+- company profile with billing information and contact person
+- list of members and pending invitations
+- address book with multiple shipping addresses
+
+![Company profile in the Back Office](img/cp_company_profile_back_office.png)
+
+From there, you can edit the company information, invite members, manage their roles and edit their basic information.
+You can also add members to a team from existing pull of users.
+Note that the Contact Person in the company has to be a member of that company.
+
+![Add Contact Person](img/cp_contact_person_back_office.png)
+
+### Admin Panel
+
+You can also manage companies from **Admin**.
+There, in the **Corporate** section, you can find a list of members,
+billing addresses and technical details regarding the organization such as visibility, IDs, relations and so on.
+
+![Corporate Back Office](img/cp_admin_company_view.png)
+
+In the **Roles** section, you can define Policies for each User Group, for example, a Company buyer.
+You can also set up Policies for every user that is using business account by editing a **Corporate Access** role.
+If you decide to remove any Policies from the **Corporate Access** role, the feature may not work properly.
```

Comment:
```suggestion
If you remove any Policies from the **Corporate Access** role, the feature may not work properly.
```
is "the feature" the whole of Corporate accounts? Then this needs rephrasing. Also, maybe put it in  a warning box?